db/seeds.rb

```diff
@@ -6,8 +6,25 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
+user = FactoryBot.create(:user)
+15.times do
+  FactoryBot.create(:street_art, user: user)
+end
```

Comment:
So tidy 😍

Reply:
So glad you implemented this. My tests could use some cleaning up!

Reply:
it won't take much time to update them now